src/lib/coda_graphql/coda_graphql.ml

```diff
@@ -1815,6 +1820,7 @@ module Mutations = struct
   let commands =
     [ add_wallet
     ; create_account
+    ; create_hd_account
```

Comment:
did you mean to add this here? Accepting regardless just pointing this out.

Reply:
The CLI doesn't build otherwise